pkg/platform/platform.go

```diff
@@ -26,19 +26,19 @@ var Platform = env.Register(
 	"PLATFORM",
 	Default,
 	"Platform where Istio is deployed. Possible values are \"openshift\" and \"gcp\"",
-).Get()
+)
```

Comment:
What's the point of this change? To do lazy eval? Under what situations would we ever need to do lazy eval for this?

Reply:
Not lazy eval. This is so this variable name (not only the value) can be used in the unit test (also part of this PR).

Reply:
You can also just do `test.SetForTest` (not `SetEnv`); a few other tests do this